src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -167,11 +160,16 @@ public function setupFieldAssociations($fields, $table, $fieldConditions, $strat
             ]);
         }
 
+        $conditions = ["$targetAlias.model" => $model];
+        if (!$this->_config['allowEmptyTranslations']) {
+            $conditions["$targetAlias.content !="] = '';
+        }
```

Comment:
Re-reading I meant to illustrate the difference between these two:
```
select * from foo where name = '';
select * from foo where name IS NULL;
```
Negating an empty string (`content!=''`) returns non-null/non-empty but when checking for an empty string (`content=''`) it'll return non-null/empty. I've been trying to get a query to break without `<>` but `!=''` seems to be doing the right thing. If I can pinpoint where/why `<>` was needed I'll bring it up again. http://stackoverflow.com/questions/14575807/how-to-select-non-empty-columns-in-mysql is a reference to null/empty checking with `<>`.